src/Dnd__DndManager.res

```diff
@@ -794,7 +794,7 @@ module Make = (Context: Context.T) => {
         let scrollPosition = Scrollable.Window.getScrollPosition()
 
         let rect = item.element->HtmlElement.getBoundingClientRect
-        let style = window->Window.getComputedStyle(item.element->Web.htmlElementToElement, _)
```

Comment:
Many functions in rescript-api rotate the `window` variable to be data-first in the function call.